mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
```

Comment:
Adding to the name of resources here isn't going to delete all existing translations - like when we update/change a file name... right? 🤔
I don't think so, but CC @mathjazz just in case

Reply:
We're good. :)